webroot/src/pages/LicensingDetail/LicensingDetail.vue

```diff
@@ -6,7 +6,103 @@
+            </div>
+            <div
+                v-for="(license, idx) in licenseeLicenses"
+                :key="idx"
+                class="license-row"
+            >
+                <div class="license-row-cell residence-state">
+                    <span v-if="$matches.phone.only" class="cell-title">{{ $t('licensing.residenceLocation') }}: </span>
+                    {{ licensee.residenceLocation() }}
+                </div>
+                <div class="license-row-cell license-state">
+                    <span v-if="$matches.phone.only" class="cell-title">{{ $t('licensing.stateOfLicense') }}: </span>
+                    {{ license.issueState.name() }}
+                </div>
+                <div class="license-row-cell practicing-state">
+                    <span v-if="$matches.phone.only" class="cell-title">{{$t('licensing.practicingLocations')}}: </span>
+                    {{ licensee.practicingLocationsAll() }}
+                </div>
+            </div>
+        </div>
+        <div class="licensee-section details">
+            <div class="licensee-section-title">Details</div>
+            <div v-if="$matches.tablet.min" class="license-row headers">
+                <div class="license-row-cell issue-date header">{{ $t('licensing.issueDate') }}</div>
+                <div class="license-row-cell expire-date header">{{ $t('licensing.expireDate') }}</div>
+            </div>
+            <div
+                v-for="(license, idx) in licenseeLicenses"
+                :key="idx"
+                class="license-row"
+            >
+                <div class="license-row-cell issue-date">
+                    <span v-if="$matches.phone.only" class="cell-title">{{ $t('licensing.issueDate') }}: </span>
+                    {{ license.issueDateDisplay() }}
+                </div>
+                <div class="license-row-cell expire-date">
+                    <span v-if="$matches.phone.only" class="cell-title">{{ $t('licensing.expireDate') }}: </span>
+                    {{ license.expireDateDisplay() }}
+                </div>
+            </div>
+        </div>
+        <div class="licensee-section adverse-actions">
+            <div class="licensee-section-title">Adverse Actions</div>
+            <div class="adverse-action-container">
+                <div class="icon"></div>
+                <div class="adverse-action">
+                    <div class="adverse-action-title">Lorem ipsum dolor</div>
+                    <div class="adverse-action-description">
+                        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse tincidunt, magna non
+                        vestibulum pulvinar, nunc felis laoreet sem, nec elementum tellus mi quis erat. Cras sit
+                        amet tellus lorem. Integer est felis, commodo a molestie id, congue sit amet lorem. Nunc
+                        pulvinar lorem id tellus ve.
+                    </div>
```

Comment:
Yep, it's in the designs as a placeholder for now. Good double check 👍

Reply:
Got it. Going forward, as we start to move towards PRs centered around user stories, I'd prefer fully-implemented code without filler data if possible (same goes for ToDo comments, commented-out code, etc).

In this case, if I understand correctly, that might mean leaving out the code for the adverse reactions until the backend is properly set up. @jusdino just FYA since this is relevant to backend code, too.

I'm fine keeping this as-is if that's easiest for now.

Reply:
@jlkravitz That makes sense. I think @jusdino & I are flexible on this topic. We'll mainly want to make sure to sync up with product / design / stakeholder folks to manage expectations & figma designs around their demos & user-research interactions.

Reply:
Makes complete sense. And one clarification after chatting with @isabeleliassen and Waldo - most important to me is that the code from each _sprint_ is complete and operational, and implements specific user stories. I'm fine if front- and back- end code is broken up into two separate PRs, e.g.